CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
+- Support for symbolic link in file mode.
```

Comment:
This should all go underneath the existing `## Added` heading below

Reply:
Fixed the `cargo fmt --all -- --check` error and updated CHANGELOG.md